lib/AI Models/AI Chatbot Seasonal Advisior/scripts/seasonal_advice.py

```
@@ -0,0 +1,103 @@
+        next_i = (i + 1) % len(season_dates)
+        next_month, next_day = season_dates[next_i]
+        
+        # Handle year wrap-around
+        if next_month < month:
+            if current_month < next_month or current_month > month:
+                return seasons[(month, day)]
+        else:
+            if month < current_month < next_month:
+                return seasons[(month, day)]
+            elif current_month == month and current_day >= day:
+                return seasons[(month, day)]
```

Ruff (0.11.9)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)